static/app/views/alerts/list/rules/row.tsx

```diff
@@ -21,11 +21,9 @@ import {Tooltip} from 'sentry/components/tooltip';
 import {IconChevron, IconEllipsis, IconUser} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {MonitorType} from 'sentry/types/alerts';
 import type {Actor} from 'sentry/types/core';
 import type {Project} from 'sentry/types/project';
 import {useUserTeams} from 'sentry/utils/useUserTeams';
-import ActivatedMetricAlertRuleStatus from 'sentry/views/alerts/list/rules/activatedMetricAlertRuleStatus';
```

Comment:
Seems like this file can likely be deleted as well?

Reply:
yeah there are a few things I'm going to clean up in a separate pr - some consts for typing and a test fixture mainly, but I'll grab this too